### PR TITLE
Update ember-cli-moment-shim version and fix tests.

### PR DESCRIPTION
**Pull request by cantsin**

### CODE REVIEW

**Comment by cantsin** [comment]
@zigahertz this is a quick fix :)

File: addon/helpers/moment-same.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import Ember from 'ember';
-import moment from 'moment';
 
 export function momentSame([date1, date2, unit]/*, hash*/) {
   if (!date1 || !date2) {
```
Comment (zigahertz) [inline addon/helpers/moment-same.js]: can we check if this guard is necessary? I don't think it is. Also - this helper can replace `moment-same-month` and `moment-same-decade`, @cantsin can you take a look at this? FYI @taras

Comment (taras) [inline addon/helpers/moment-same.js]: Yes, we can remove `moment-same-decade` and `moment-same-month` I believe. We also need to PR this helper into ember-moment. 

What does this shim do?

**Comment by zigahertz** [comment]
@taras good point re: PR for `ember-moment`, I'll do this now

**Comment by cantsin** [comment]
@zigahertz the check is required -- otherwise we get `Cannot read property 'isSame' of null`. Nonetheless, I have simplified it somewhat, using a tertiary operator.

I cannot get rid of `same-decade` -- this seems required. `isSame` from the moment docs do not seem to support `'decade'` as a parameter. `same-month` is gone.

**Comment by zigahertz** [comment]
lgtm!

**Pull request merged**